src/mmw/js/src/compare/templates/compareModeling.html

```diff
@@ -0,0 +1,19 @@
+<h3>Model Output</h3>
+{% if polling %}
+    <i class="fa fa-circle-o-notch fa-spin"></i>
+{% endif %}
+<div class="pull-right">
+    <select class="form-control btn btn-small btn-primary">
+        {% for result in results %}
+            {% if result.active %}
+                <option value="{{ result.name }}" selected>
+            {% else %}
+                <option value="{{ result.name }}" >
+            {% endif %}
```

Comment:
This entire `if` block can probably be simplified using an [`if` expression](https://mozilla.github.io/nunjucks/templating.html#if-expression):

```jinja
<option value="{{ result.name }}" {{ "selected" if result.active }}>
```